tools/test_format.py

```diff
@@ -8,11 +9,11 @@
 
 
 def main():
-    util.run([sys.executable, "tools/format.py"])
+    util.run(["target/release/deno", "--allow-run", "tools/format.ts"])
```

Comment:
1. I'm not sure how this works on Windows, because it should need the `.exe` extension.  Use `executable_suffix` in `util.py` 
2. In certainly situations we might have `target/debug/deno` and not the release one. It would be good to check for existence of each, preferring the release one.

Reply:
I added code to choose deno in preference of `release` -> `debug` -> globally installed version.